prow/config.yaml

```diff
@@ -15330,7 +15330,7 @@ periodics:
         value: /etc/aws-ssh/aws-ssh-private
       - name: JENKINS_AWS_SSH_PUBLIC_KEY_FILE
         value: /etc/aws-ssh/aws-ssh-public
-      image: gcr.io/k8s-testimages/kubekins-e2e:v20171023-364289b6-master
+      image: gcr.io/k8s-testimages/kubekins-e2e:latest-master
```

Comment:
can you add `imagePullPolicy: Always` as well?

Reply:
Good point - thanks!  Updated!

Out of interest / penance - do we have a tester / linter for bad prow configs I could add this check to?